frontend/src/__tests__/unit/utils/time.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { convertPeriodicTimeToSeconds } from '~/utilities/time';
+
+describe('Convert periodic time to seconds', () => {
+  it('should convert hours to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('5Hour')).toBe(5 * 3600);
+  });
+
+  it('should convert minutes to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('6Minute')).toBe(6 * 60);
+  });
+
+  it('should convert days to seconds', () => {
+    expect(convertPeriodicTimeToSeconds('221Day')).toBe(221 * 86400);
```

Comment:
You broke up a week (`7 * 24 * 60 * 60`) but just used a flat magic number for day?